.github/workflows/unitTests.yml

```diff
@@ -0,0 +1,25 @@
+name: Run unit tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: sbt test
+      - name: Check output
+        run: |
+          echo "Pull Request Number - ${{ steps.cpr.outputs.pull-request-number }}"
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Formatting
+        run: sbt scalafmtSbtCheck scalafmtCheck test:scalafmtCheck
```

Comment:
Nit: add new line

Reply:
what's the particular reason for adding new line btw? I never asked before, but figured I'd ask since you seem to suggest newline all the time

Reply:
Good question. In addition to the reasons [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file), I also personally find the red marking GitHub adds to diffs distracting  🤷‍♂️